sale_secondary_salesperson/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+There is no need to do anything to configure this module
```

Comment:
This file is not necessary if this module doesn't need any kind of configuration.